CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Handle NPE in GetResult if "found" field is missing ([#14552](https://github.com/opensearch-project/OpenSearch/pull/14552))
 - Refactoring FilterPath.parse by using an iterative approach ([#14200](https://github.com/opensearch-project/OpenSearch/pull/14200))
 - Refactoring Grok.validatePatternBank by using an iterative approach ([#14206](https://github.com/opensearch-project/OpenSearch/pull/14206))
+- update help output for _cat ([#14722](https://github.com/opensearch-project/OpenSearch/pull/14722))
```

Comment:
```suggestion
- Update help output for _cat ([#14722](https://github.com/opensearch-project/OpenSearch/pull/14722))
```
(that is going to be user facing release notes) ;-)

Reply:
haha noted 😄